lib/validateMessage.js

```diff
@@ -153,16 +153,18 @@ function validateScope(isValid, scope) {
       error('a scope is required !');
       isValid = false;
     }
-    if (isValid && multipleScopesAllowed) {
-      scopes.forEach(validateIndividualScope);
-    }
-    if (isValid && !multipleScopesAllowed) {
-      if (scopes.length > 1) {
-        error('only one scope can be provided !');
-        isValid = false;
+    if (scopes.length > 0) {
```

Comment:
+1 to the early return as @spirosikmd recommended here. It is currently at 3 levels of if nesting, this makes 4 levels which is quite difficult to follow through. Having the early return keeps the nesting at 3 levels which is basically as minimal as we can get.